relay.go

```diff
@@ -675,7 +675,7 @@ func (r *Relayer) finishRelayItem(items *relayItems, id uint32) {
 }
 
 func (r *Relayer) decrementPending() {
-	r.pending.Dec()
+	r.pending.Add(^uint32(0))
```

Comment:
```suggestion
	r.pending.Add(-1)
```
right?

Reply:
Unfortunately pending is unsigned, & there's no atomic subtract method, so this is the documented way: https://pkg.go.dev/sync/atomic#AddUint32